src/coreComponents/physicsSolvers/simplePDE/LaplaceVEM.cpp

```diff
@@ -0,0 +1,448 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file LaplaceVEM.cpp
+ */
+
+// Source includes
+#include "LaplaceVEM.hpp"
+
+#include "mpiCommunications/CommunicationTools.hpp"
+#include "common/TimingMacros.hpp"
+#include "common/DataTypes.hpp"
+#include "virtualElement/ConformingVirtualElementOrder1.hpp"
+#include "managers/DomainPartition.hpp"
+#include "managers/NumericalMethodsManager.hpp"
+#include "managers/GeosxState.hpp"
+
+namespace geosx
+{
+
+namespace dataRepository
+{
+namespace keys
+{}
+}
+
+using namespace dataRepository;
+
+/*----------------------------------------------------------------------------------
+ * LaplaceVEM: Solving Laplace's partial differential equation with virtual elements
+ * ---------------------------------------------------------------------------------
+ *
+ * What does this solver do?
+ * --------------------------
+ *
+ * This solver finds a solution f(x,y,z) to the Laplace equation: div ( grad ( f )) = 0
+ * This common elliptic PDE represents the solution of a steady-state heat transfer, for instance.
+ *
+ * Where can I find an example of what it does?
+ * --------------------------------------------
+ *
+ * Integrated tests associated to this solver are found in the ./integratedTests/ folder
```

Comment:
@andrea-borio, Does this actually describe your integrated tests?  This header looks like cut and paste from the FEM version, so we should make sure it is still accurate.

Reply:
It is indeed a cut and paste of the text that you wrote in `LaplaceFEM.cpp`, but I changed all mentions to finite elements into mentions to virtual elements. The text is valid, it tells the truth. When we write a base class, we will have just one.